city_scrapers/spiders/chi_police_retirement.py

```diff
@@ -0,0 +1,129 @@
+        year = self._parse_year(response)
+        date_table = response.xpath('//*[@id="content0"]/div[3]/table')
+        date_items = date_table.extract()[0].split('<tr>')
+
+        skip_first_item = True
+        for item in date_items:
+            if skip_first_item:
+                skip_first_item = False
+                continue
+
+            meeting = Meeting(
+                title=self._parse_title(response),
+                description=self._parse_description(response),
+                classification=self._parse_classification(response),
+                start=self._parse_start(item, year),
+                time_notes=self._parse_time_notes(),
+                end=self._parse_end(),
+                all_day=False,
+                location=self._parse_location(response),
+                source=self._parse_source(response),
+                links=self._parse_links(item)
+            )
+
+            # replace before this
+            meeting["status"] = self._get_status(meeting)
+            meeting["id"] = self._get_id(meeting)
+
+            yield meeting
+
+    def _parse_title(self, item):
+        """Parse or generate meeting title."""
+        return "Board Meetings of The Policemen\'s Annuity & Benefit Fund"
+
+    def _parse_description(self, response):
+        """Parse or generate meeting description."""
+        raw_description = ''.join(response.xpath('//*[@id="content0"]/p/text()').extract())
+        clean_description = self._clean_escape_chars(raw_description)
+        print(repr(' '.join(clean_description.split())))
+        return ' '.join(clean_description.split())
+
+    def _parse_classification(self, item):
+        """Parse or generate classification from allowed options."""
+        return BOARD
+
+    def _parse_start(self, item, year):
+        start = self._get_date_string(item, year)
+        return datetime.strptime(start, '%B %d %Y %I%p')
+
+    def _parse_time_notes(self):
+        return None
+
+    def _parse_end(self):
+        return None
+
+    # see here for address: http://www.chipabf.org/ChicagoPolicePension/aboutus.html
+    def _parse_location(self, item):
+        """Parse or generate location."""
+        return {
+            "address": "221 North LaSalle Street, Suite 1626,"
+                       " Chicago, Illinois 60601-1203",
+            "name": "Policemen's Annuity and Benefit Fund",
+        }
+
+    def _parse_source(self, response):
+        """Parse or generate source."""
+        return response.url
+
+    def _parse_links(self, item):
+        links = []
+        raw_links = item.split('a href')
+        if len(raw_links) > 1:
+            raw_agenda = raw_links[1]
+            file_path = re.search(r'\"(.+?)\"', raw_agenda).group(1)
+            title = re.search(r'\>(.+?)\<', self._clean_escape_chars(raw_agenda)).group(1).strip()
+            agenda = {
+                "href": 'https://' + self.allowed_domains[0] + '/ChicagoPolicePension/' + file_path,
```

Comment:
An easier way to do this is to pass the `response` object as an argument to this method and then call `response.urljoin(file_path)` for the `href` value.

It uses the same logic as the browser does when opening the link to create the full path and keeps things a bit simpler. This also goes for the `href` in minutes